modules/ROOT/pages/policy-mule3-add-remove-headers.adoc

```diff
@@ -136,43 +125,43 @@ You can use wildcards to remove properties that have similar names. For example,
 
-== Testing Policy Applications
+// == Testing Policy Applications
 
-To test the execution of policies, follow the steps in this section to apply a policy and use curl, for example `curl` `+http://localhost:8081 -vvv+` to test the policies.
+// To test the execution of policies, follow the steps in this section to apply a policy and use curl, for example `curl` `+http://localhost:8081 -vvv+` to test the policies.
 
-=== Add Request Header Policy
+// === Add Request Header Policy
 
-The procedure for testing the add or remove request header policies assume that you created a mule application that logs the headers received by the backend service.
+// The procedure for testing the add or remove request header policies assume that you created a mule application that logs the headers received by the backend service.
 
-. Apply the add request header policy to add a header as described in the previous section.
-. Run curl.
-. Check the log of the backend service to verify that request includes the additional header name and value you configured in step 1.
+// . Apply the add request header policy to add a header as described in the previous section.
+// . Run curl.
+// . Check the log of the backend service to verify that request includes the additional header name and value you configured in step 1.
 
 
-=== Remove Request Header Policy
+// === Remove Request Header Policy
 
-The procedure for testing the remove request header policy assumes that you created a mule application that logs the headers received by the backend service.
+// The procedure for testing the remove request header policy assumes that you created a mule application that logs the headers received by the backend service.
 
-. Apply the remove request header policy to remove a header as described in the previous section.
-. Run curl.
-. Verify that the backend service excludes the extra header by checking the log.
+// . Apply the remove request header policy to remove a header as described in the previous section.
+// . Run curl.
+// . Verify that the backend service excludes the extra header by checking the log.
 
 
-=== Add Response Header Policy
+// === Add Response Header Policy
 
-. Run curl.
-. Check the response to see which headers are being received by the client.
-. Apply the add response header policy to add a response to the header.
-. Run curl again.
-. Verify that the specified header is received.
+// . Run curl.
+// . Check the response to see which headers are being received by the client.
+// . Apply the add response header policy to add a response to the header.
+// . Run curl again.
+// . Verify that the specified header is received.
 
-=== Remove Response Header Policy
+// === Remove Response Header Policy
 
-. Run curl.
-. Check the response to see which headers are being received by the client.
-. Apply the remove response header policy to remove a response from the header.
-. Run curl again.
-. Verify that the specified header is removed.
```

Comment:
Ditto previous edits to the similar steps above

Reply:
Text removed! Thanks!